compiler/optimizer/OSRDefAnalysis.cpp

```diff
@@ -886,7 +1081,7 @@ int32_t TR_OSRLiveRangeAnalysis::perform()
    block = comp()->getStartBlock();
    while (block)
       {
-      memset(_pendingSlotValueParents, 0, numBits*sizeof(NodeParentInfo *));
+      //memset(_pendingSlotValueParents, 0, numBits*sizeof(NodeParentInfo *));
```

Comment:
please remove deadcode, don't just comment it

Reply:
Rather than disabling the copy propagation allocations, it has been removed entirely, addressing the four comments concerning it.